Controller/SectionController.php

```diff
@@ -46,11 +46,13 @@ public function listAction()
     {
         $sectionList = $this->sectionService->loadSections();
         $contentCountBySectionId = [];
+        $canDeleteBySectionId = [];
         $deleteFormsBySectionId = [];
 
         foreach ($sectionList as $section) {
             $sectionId = $section->id;
             $contentCountBySectionId[$sectionId] = $this->sectionService->countAssignedContents($section);
+            $canDeleteBySectionId[$sectionId] = !$this->sectionService->isSectionUsed($section);
```

Comment:
The section list doesn't have pagination yet, so with many sections it can be a problem. But that's the case with or without this PR, and it's a separate concern, I think.